README.md

```diff
@@ -48,8 +50,83 @@ const { useInitZkStore, useZKStore, useGetLatestProof } =
             num: state.num + 1,
           };
         }),
-    })
+    }),
+    
+    // Define the state variable that you want to trigger a proof generation on
+    // NOTE Functions will never trigger a proof generation
+    ["num"]
   );
 ```
 
-<!-- TODO: properly document functions -->
\ No newline at end of file
+## Configuring your project
+
+To enable SnarkyJS for the web, we must set the COOP and COEP headers. When using a Vite project we also need to install a plugin to enable topLevelAwait for the web worker.
```

Comment:
It may be worth mentioning here that zkStates leverages SnakryJS for proof generation on the browser. So that people understand why we even mention SnarkyJS at all.